bin/cron/calculate_workshop_survey_results

```diff
@@ -15,6 +17,24 @@ def main
     course_scores[course] = get_score_for_workshops(workshops)
   end
 
+  course_scores['CSP Local Summer Workshops'] = summarize_workshop_surveys(
+    Pd::Workshop.in_state(Pd::Workshop::STATE_ENDED).where(course: Pd::Workshop::COURSE_CSP, subject: Pd::Workshop::SUBJECT_CSP_SUMMER_WORKSHOP),
+    nil,
+    false
```

Comment:
Should these be named parameters on `summarize_workshop_surveys` so it's more clear? Then also you would only need to specify those differ from the defaults.